src/mlm_client_engine.inc

```diff
@@ -46,16 +46,17 @@ typedef enum {
     set_worker_event = 7,
     ok_event = 8,
     expired_event = 9,
-    stream_deliver_event = 10,
-    mailbox_deliver_event = 11,
-    service_deliver_event = 12,
-    error_event = 13,
-    connection_pong_event = 14,
-    replay_ready_event = 15,
-    heartbeat_event = 16,
-    command_invalid_event = 17,
-    failed_event = 18,
-    other_event = 19
+    bad_pattern_event = 10,
```

Comment:
This code is autogenerated by zproto, and I am not aware of such functionality there :(